tests/tests_ethernet_nm.yml

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please keep the actual test in one file that is included by the specific files for initscripts and nm as done in https://github.com/linux-system-roles/network/blob/master/tests/tests_ethtool_features_initscripts.yml